cpp/data/installer/scripts/install_centos_packages.sh

```diff
@@ -41,7 +41,7 @@ dnf --assumeyes update
 # basic dependencies
 InstallIfMissing "ca-certificates"
 dnf --assumeyes install \
-    ant bc cifs-utils clang crontabs ftp gawk gcc-c++ git glibc-all-langpacks ImageMagick incron java-1.8.0-openjdk-devel \
```

Comment:
Please add a shell script for existing systems that installs fetchmail (e.g. system_updates/2.sh)

Reply:
I will, eventually.

Reply:
Otherwise i will "eventually" not merge the pull request :stuck_out_tongue_winking_eye: